src/api/cli/create_rawtx.go

```diff
@@ -531,14 +531,31 @@ func makeChangeOut(outs []wallet.UxBalance, chgAddr string, toAddrs []SendAmount
 		return nil, err
 	}
 
-	if haveChange {
-		outAddrs = append(outAddrs, mustMakeUtxoOutput(chgAddr, changeAmount, changeHours))
-	}
-
+	var spendCoinsAmt uint64
 	for i, to := range toAddrs {
+		// check if changeHours > 0
```

Comment:
Can you put a longer comment describing the purpose of this added code (to send maximum of coin hours == coins, minimum of 1)